### ISSUE TITLE
feat: allow exclusing of targets

### ISSUE THREAD

**Issue by strowi** [grafana/tanka#801]
Hi,

i noticed during testing i often have to delete bigger parts of the k8s-stack. After doing this for a while i noticed that letsencrypt was running into rate-limits, because the ingress would be deleted too.
It would be nice to have the opposite of "-t" to be able to specify excluded resources.

**Comment by zerok** [comment]
Hi :) What's your actual workflow? Do you run `tk apply`, then do manual changes, and combine that with something like `tk delete`?

**Comment by strowi** [comment]
Hey ;)

Been a while since i wrote that, but I "think" the actual problem here was me being lazy and wiping ALL secrets (with `kubectl delete ...,secrets --all` ) from a namespace (because some operator would create those automatically). And letsencrypt was more or less a bad side-effect.

Also it seems i can do what i want with negative-regex (`tk apply .. -t '!...'`) although a little more complicated).

**Comment by zerok** [comment]
Yes, that should do it 🙂  I'll close this issue then. If it's still relevant, please reopen 🙂
